images/ig-publisher/Dockerfile

```diff
@@ -0,0 +1,90 @@
+FROM openjdk:23-jdk-slim-bookworm as base
+LABEL maintainer="Josh Mandel"
+
+ARG NODE_VERSION=v20.13.1
+
+# Install dependencies, clean up, and set up non-root user
+RUN apt-get update && apt-get -y install curl git openssl wget graphviz ruby ruby-dev build-essential libffi-dev && \
+    apt-get clean && \
+    gem install jekyll jekyll-asciidoc
+
+# Download and install Node.js
+WORKDIR /tmp
+
+# Set platform-specific variables
+RUN if [ -z "$NODE_DISTRO" ]; then \
```

Comment:
You might wanna do something like this [https://docs.docker.com/build/building/multi-platform/ instead](https://docs.docker.com/build/building/multi-platform/#cross-compilation) instead

Reply:
Agreed, but I'm going to leave this out of scope for now. Let's land these changes (which should be buildable/runnable on linux + mac) and then think about cross-platform builds. (Especially since I can't test them with my current setup.)